### PR TITLE
Render HTML in description fields

### PR DESCRIPTION
**Pull request by michaelglenister**
## Description

These changes will render HTML elements for descriptions from the backend

## Related Issue
https://trello.com/c/E9XN4wrh/803-rich-text-descriptions-as-a-user-i-would-like-to-be-able-to-click-links-in-descriptions-and-read-them-easily-so-that-i-can-more

### Added
Description fields can now display HTML set by data admins

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design matches the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [x]  ✅ added (appropriate) unit tests
- [x]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

**Comment by emre2038** [comment]
@michaelglenister `mapchip` test looks good to me. could you also add tests for `profile/category` and `profile/subcategory`?

File: __tests__/elements/mapchip.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    })
+
+    test('Description is visible and renders HTML tags', () => {
+      let component = new Component();
+      let mc = new MapChip(component, mapchip_colors)
+      mc.show();
+      mc.description = mapchip_args.data.description;
+      
+      expect(descriptionField).toBeVisible();
+      expect(descriptionField.textContent.trim()).toBe('An HTML description')
```
Comment (adieyal) [inline __tests__/elements/mapchip.test.js:39]: Does this test if it is displaying html or only text?

If not, perhaps something like
```
...
<p>An <strong>HTML</strong> description</p>
...
htmlTag = descriptionField('strong);
...
expect(htmlTag).not.toBeNull()
```

Comment (michaelglenister) [reply __tests__/elements/mapchip.test.js]: It's checking that the HTML tags are rendered and do no appear the text e.g. <p>An <strong>HTML</strong> description</p> would then just be 'An HTML description'. I did some cleaning of the test, but it's important to check the content of htmlTag after being set

Comment (adieyal) [reply __tests__/elements/mapchip.test.js]: but how do you know that it's adding html not escaped text? Wasn't that the bug that was being addressed in the first place?
